server/data/restClientMetricsMiddleware.test.ts

```diff
@@ -47,13 +47,13 @@ describe('restClientMetricsMiddleware', () => {
         .set('accept', 'json')
         .then(res => {
           code = res.statusCode
-        })
 
-      expect(code).toBe(200)
-      expect(requestHistogramLabelsSpy).toHaveBeenCalledTimes(1)
-      expect(requestHistogramLabelsSpy).toHaveBeenCalledWith('httpbin.org', 'GET', '/', '200')
-      expect(requestHistogramStartSpy).toHaveBeenCalledTimes(1)
-      nock.cleanAll()
+          expect(code).toBe(200)
+          expect(requestHistogramLabelsSpy).toHaveBeenCalledTimes(1)
+          expect(requestHistogramLabelsSpy).toHaveBeenCalledWith('httpbin.org', 'GET', '/', '200')
+          expect(requestHistogramStartSpy).toHaveBeenCalledTimes(1)
+          nock.cleanAll()
+        })
```

Comment:
This one is interesting given the expectations shouldn't run until we've finished (a)waiting. I guess there's a possibility that the chained operations fail somewhere and don't get to setting the code?

Reply:
I've redone this and used the `await`ed value which I think feels more readable here [dc8180f](https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/pull/62/commits/dc8180f30781ed5acc311e3e511be7d03a43be6d)

Reply:
Nice!